tests/manage/test_ocs_288.py

```diff
@@ -73,12 +78,15 @@ def create_multiple_rbd_storageclasses(count=1):
          count (int): count specify no of storageclass want to create by
             default count is set to one i.e it will create one sc
     """
+    global POOL_OBJ, SC_RBD_OBJ
+    POOL_OBJ = [0] * count
+    SC_RBD_OBJ = [0] * count
     for sc_count in range(count):
         log.info("Creating CephBlockPool")
-        pool_obj = helpers.create_ceph_block_pool()
-        helpers.create_storage_class(
+        POOL_OBJ[sc_count] = helpers.create_ceph_block_pool()
```

Comment:
This is mutating the state of this global variable``POOL_OBJ``. Did you mean to  deepcopy ``POOL_OBJ`` before using it?

Reply:
didn't get you